fasteners/process_lock.py

```diff
@@ -214,30 +214,44 @@ def exists(self):
         return os.path.exists(self.path)
 
     def trylock(self):
-        raise NotImplementedError()
+        self._trylock(self.lockfile)
 
     def unlock(self):
+        self._unlock(self.lockfile)
+
+    @staticmethod
```

Comment:
It could be done a few different ways. What I needed was a way to access these methods from the tests while being able to pass in my own lockfile handle. They could have also been standalone functions, but since the code was already making sure that `InterProcessLock` had the correct implementation per platform, it seemed easiest to just stick with the inheritance route. They're static because the don't depend on anything and because it lets you use them without creating an instance with a dummy lock file as `__init__` would require.